recipe/meta.yaml

```diff
@@ -1,66 +1,64 @@
-{% set version = "6.1.0rc1" %}
+{% set version = "7.0.0b4" %}
```

Comment:
Looks like this published `7.0.0b4` on the rc channel:

![image](https://github.com/conda-forge/notebook-feedstock/assets/591645/4d097786-2afe-4569-8a58-c3a48c7a4f25)

Doing the update to rc1 directly in https://github.com/conda-forge/notebook-feedstock/pull/119